src/main/java/com/github/manolo8/darkbot/utils/login/LoginUtils.java

```diff
@@ -210,4 +247,13 @@ public WrongCredentialsException(String s) {
         }
     }
 
+    public static class LoginURLException extends  WrongCredentialsException {
```

Comment:
What i ment was the oposite:

have a `LoginException` that is the more generic one
then have `WrongCredentialsException extends LoginException`

Then make the code catching WrongCredentialsException catch the more generic LoginException, although i think that code already catches Exception so it should be fine